docs/general/builtins/scalar-functions.rst

```diff
@@ -1462,6 +1462,42 @@ specified interval added to the timestamp of ``0000/01/01 00:00:00``::
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-age:
+
+``age(['timezone',][timestamp,]timestamp)``
+-------------------------------------------
+
+Returns: ``bigint``
+
+The ``age`` function can be used to calculate the difference between timestamp inputs
+or timestamp input and today at midnight if is called with one timestamp parameter.
+
+The return value is the age in milliseconds.
+
+It is implemented to improve compatibility with
+clients that use the PostgreSQL wire protocol.
+
+If at least one argument is ``NULL``, the return value is ``NULL``.
+
+Valid values for ``timezone`` are either the name of a time zone (for example
```

Comment:
I'm not sure whether we need timezone for that function - PG supports only 1 or 2 timestamps without timezone as arguments - for T1, T2 at the same timezone age would be same as at UTC - since age is just difference, ie
utcT2 - utcT1 = utcT2 + offset - (utcT1  + offset). 
Same for one arg: utcT - utcOrigin_which_is_actually_0 = utcT + offset - (utcOrigin_which_is_actually_0 + offset)

Reply:
Obviously this is the same. And initially i thought that this doesn't make sense in postgres.And i filled that only to be the same as date trunc 
and because postgres can accept timstamp with timezone everywhere.

But as i can check in postgres:

postgres=# select age(timestamp with time zone '2020-06-30 02:00:00+01');
              age               
--------------------------------
 1 year 4 mons 10 days 23:00:00
(1 row)

postgres=# select age(timestamp '2020-06-30 02:00:00');                  
              age               
--------------------------------
 1 year 4 mons 10 days 22:00:00
(1 row)

With midnight always considered to UTC and not the same timezone as the input parameter that i initially assumed. 

So should i remove the timezone parameter or change code in order to have always midnight in UTC like Postgres? The first option is simpler... i don't know if this is important about postgres compatibility

Reply:
Let's just drop timezone